content/guides/hot-module-replacement.md

```diff
@@ -94,6 +88,13 @@ __index.js__
   }
 
   document.body.appendChild(component());
++
```

Comment:
Ah or maybe you meant there's two plus signs? That's actually just a looking at a `diff` within a `diff` issue 😆  (there's really only one in the document).